tests/utils.go

```diff
@@ -1435,3 +1437,24 @@ func GetNodeWithHugepages(virtClient kubecli.KubevirtClient, hugepages k8sv1.Res
 	}
 	return nil
 }
+
+// SkipIfVersionBelow will skip tests if it runs on an environment with k8s version below specified
+func SkipIfVersionBelow(message string, expectedVersion string) {
+	virtClient, err := kubecli.GetKubevirtClient()
+	PanicOnError(err)
+
+	response, err := virtClient.RestClient().Get().AbsPath("/version").DoRaw()
+	Expect(err).ToNot(HaveOccurred())
+
+	var info k8sversion.Info
+
+	err = json.Unmarshal(response, &info)
+	Expect(err).ToNot(HaveOccurred())
+
+	curVersion := strings.Split(info.GitVersion, "+")[0]
+	curVersion = strings.Trim(curVersion, "v")
+
+	if curVersion < expectedVersion {
```

Comment:
On second thought, I'm not entirely comfortable with this. This line is doing a lexical comparison for "less than" on semantic version strings. If almost feels like it gets the right answer by accident.

Reply:
@stu-gott I discussed it with Roman, string comparison convert each character to ASCII and compare it, so until we do not changes under ASCII table all must work as expected :smile:

Reply:
The comment here is also helpful in this context: https://golang.org/src/strings/compare.go